invenio_oaiserver/utils.py

```diff
@@ -79,3 +79,42 @@ def datetime_to_datestamp(dt, day_granularity=False):
     if day_granularity:
         result = result[:-10]
     return result
+
+
+def eprints_description(content, metadataPolicy, dataPolicy, submissionPolicy):
```

Comment:
The "content" and "submissionPolicy" should be optional: https://www.openarchives.org/OAI/1.1/eprints.xsd

Passing empty string for those would not be nice either since it would format multiple newlines, and more generally: perhaps we should use some more sophisticated XML element building tool like  ``lxml.builder.E`` (also for friends and oai-identifier)

Reply:
Good point, thanks for the suggestion 👍